manual/manual/refman/exten.etex

```diff
@@ -241,18 +241,18 @@ The typical use of private types is in the export signature of a
 module, to ensure that construction of values of the private type always
 go through the functions provided by the module, while still allowing
 pattern-matching outside the defining module.  For example:
-\begin{verbatim}
-        module M : sig
-                     type t = private A | B of int
-                     val a : t
-                     val b : int -> t
-                   end
-                 = struct
-                     type t = A | B of int
-                     let a = A
-                     let b n = assert (n > 0); B n
-                   end
-\end{verbatim}
+\begin{caml_example*}{verbatim}
+module M : sig
+             type t = private A | B of int
+             val a : t
+             val b : int -> t
+           end
+         = struct
+             type t = A | B of int
+             let a = A
+             let b n = assert (n > 0); B n
+           end
+\end{caml_example}
```

Comment:
May I suggest to add a star here `\end{caml_example*}` to not betray the illusion that caml_example is a latex environment? (pure nitpicking, I wholeheartedly admit)